.github/workflows/regression_tests_docker.yml

```diff
@@ -0,0 +1,56 @@
+name: Run Regression Tests on Docker
+
+on:
+  push
+    # run every day at 5:15am
+  #schedule:
+  #  - cron:  '15 5 * * *'
+
+concurrency:
+  group: ci-cpu-${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  docker-regression:
+    strategy:
+      fail-fast: false
+      matrix:
+        hardware: [ubuntu-20.04, [self-hosted, regression-test-gpu]]
+    runs-on:
+      - ${{ matrix.hardware }}
+    steps:
+      - name: Clean up previous run
+        run: |
+          echo "Cleaning up previous run"
+          ls -la ./
+          sudo rm -rf ./* || true
+          sudo rm -rf ./.??* || true
+          ls -la ./
+          docker system prune -f
+      - name: Checkout TorchServe
+        uses: actions/checkout@v3
+      - name: Branch name
+        run: |
+          echo $GITHUB_REF_NAME
+      - name: Build CPU Docker Image
```

Comment:
Is there now enough memory on non custom runners to build both images? My main worry is that subsequent steps should fail if this step fails

Reply:
These will be executed on 2 different runners. One for CPU and one for GPU